Keeran10/CurriculumManagementSystem
# Add test coverage report for Angular

We were missing coverage report for front-end tests.

To verify, do the following:
1) `ng test` or `npm run test`
2) a new folder named `coverage` should appear in `cms-client`.
3) right-click on `coverage/index.html`.
4) choose `open with live server`.

You should see this report opened on your web browser:
![image](https://user-images.githubusercontent.com/19395031/71836594-51887300-3082-11ea-96f9-6220a09b69c0.png)


This PR completes issue #153.
